treepoem/__main__.py

```diff
@@ -0,0 +1,42 @@
+# -*- encoding:utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import argparse
+import sys
+from textwrap import fill
+
+from . import generate_barcode
+from .data import barcode_types
+
+supported_barcode_types = (
+    'Supported barcode types are:\n'
+    + fill(', '.join(sorted(barcode_types)), initial_indent='    ', subsequent_indent='    ')
+)
+
+
+if sys.version_info < (3, 0):
+    stdout_binary = sys.stdout
+else:
+    stdout_binary = getattr(sys.stdout, 'buffer')
```

Comment:
you don't need `getattr` here, just use a plain dotted access

Reply:
The reason I thought you might prefer `getattr` is that _static_ code analysis tools (e.g. rope) will squawk about this line when looking at it in "Python 2.x" mode.

If you don't care about that, I'll change it.